mirth.rb

```diff
@@ -23,8 +23,7 @@ def show_data
 
   def add_data
     DailyData.create(date: params['date'], step_count: params["step_count"], notes: params["notes"])
-    redirect_to("/show/data")
-    render(status: :see_other)
+    redirect_to("/show/data", status: 303)
```

Comment:
This is what you had before, but I was [trying to say](https://github.com/wildmaples/mirth/pull/10#discussion_r581303492) that you can use `:see_other` instead of `303`.

Reply:
[LOL](https://github.com/wildmaples/mirth/pull/10#discussion_r581368734) but thanks, I get it now